components/advanced_create_post/advanced_create_post.test.jsx

```diff
@@ -1166,6 +1166,7 @@ describe('components/advanced_create_post', () => {
                 message: '/fakecommand some text',
             }),
             expect.anything(),
+            expect.anything(),
```

Comment:
why is this being duplicated?

Reply:
I added argument.
https://github.com/mattermost/mattermost-webapp/pull/11370/files#diff-4e310fd11a2a4cd1657309cfabb0bc7d4254ec16851c42a6162c7460c24b7afaR185